docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+
+version: '3.7'
+
+services:
+    github-graphs:
+        build:
+            context: .
+            dockerfile: Dockerfile
```

Comment:
This is fine, but in general if the context is the same directory, you don't need to give context, as it looks for a `Dockerfile` in the root by default.